vsts/scripts/pullandretag.sh

```diff
@@ -6,22 +6,55 @@
 
 set -o pipefail
 
+declare integrationtestfilter="oryxdevmcr.azurecr.io/public/oryx/"
```

Comment:
This can change, it'll probably be more robust to use the value from `__variables.sh`

Reply:
IMO we should keep vsts build scripts and source code independent. Though we can use some variables by sourcing variables.sh, its better not to do it here. If the name changes, we need to change it everywhere including yaml templates.